app/src/main/res/layout/activity_main.xml

```diff
@@ -1,20 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mataku/SunsetScrob/blob/358f4d3d5d1451906a8d4af2536fd5c14fbe83c0/app/src/main/res/layout/activity_main.xml#L2"></span>The resource <code>R.layout.activity_main</code> appears to be unused</td>
    </tr>
  </tbody>
</table>